Tests/iaas/scs-0103-v1-flavors.yaml

```diff
@@ -0,0 +1,192 @@
+meta:
+  name_key: name-v2
+flavor_groups:
+  - status: mandatory
+    list:
+    - name: SCS-1V-4
```

Comment:
One of the design principles in the old way of listing Mandatory (and Recommended) flavors was to NOT keep long lists of settings in a manually managed YAML file.
Instead we only had the names and had code that derived all settings from the name to generate the verbose form. This had the big advantage to keep the human-managed data short and avoid any divergence between the settings encoded in the file. I would consider a list here with just names. The amount of `cpus`, `ram` as well as `disk1 and `cpu-type` and `name-v1` and `name-v2` should be extracted from the name and generated by code.

The other option would be to have tooling that is included in our CI that validates the input for consistency. I would consider it less preferable over the condensed format, but would not veto it, as the ones who do the main work obviously have to feel comfortable with the design and tooling.

Reply:
fwiw I tend to agree with @mbuechse here, because I find it cleaner to separate code - which does the checks - and data - the flavors to be checked -.

As a middle ground to keep the line count lower I'd suggest the usage of yaml anchors and aliases like this:

```yaml
  - foo: &SCS-2V
    - name: SCS-2V-8
      cpus: 2
      cpu-type: shared-core
      ram: 8
  - foo: *SCS-2V
    - name: SCS-2V-4-20s
      ram: 4
      disk: 20
  - foo: *SCS-2V
    - name: SCS-2V-16
      ram: 16
```

Reply:
> I strongly oppose deriving the metadata from the name. One of the main principles behind my approach is separation of concerns, that is, avoiding coupling where possible:
> 
> - From the point of view of this standard, the names are opaque/arbitrary.
> - It is up to the naming standard to check whether the properties and the name are consistent. (Which the script for that standard actually does, except for the extra_specs.)

Fair enough.
Some consistency checks should be done as CI job then I believe, because invariably we will screw up sooner or later otherwise.

Reply:
And I'm fine with pushing this consistency check out into a new issue, so we can get this PR merged now.